animeal/src/Flows/Profile/ProfileViewController.swift

```diff
@@ -67,7 +67,8 @@ final class ProfileViewController: BaseViewController, ProfileViewable {
         navigationItem.hidesBackButton = viewConfiguration.hidesBackButton
         if !viewConfiguration.hidesBackButton {
             navigationItem.hidesBackButton = true
-            let backButton = UIBarButtonItem(image: UIImage(named: Asset.Images.arrowBackOffset.name), style: .plain, target: self, action: #selector(backTapped))
+            let img = UIImage(named: Asset.Images.arrowBackOffset.name)
```

Comment:
nit: variable names should be meaningful

Reply:
Done